util/src/main/scala/geotrellis/util/StreamBytes.scala

```diff
@@ -0,0 +1,22 @@
+package geotrellis.util
+
+import java.io.InputStream
+import java.nio.ByteBuffer
+
+abstract class StreamBytes(chunkSize: Int) {
```

Comment:
This also needs docstrings, since its hard to tell at a glance if this is a "noun" or a "verb" class.